Cargo.toml

```diff
@@ -13,7 +13,7 @@ keywords = ["mongo", "mongodb", "database", "bson", "nosql"]
 license = "Apache-2.0"
 
 [dependencies]
-bson = "0.3"
+bson = "0.3.2"
```

Comment:
Looking back, I'm not actually sure why we didn't specify a release for our dependencies. (I'm guessing that I probably thought that specifying the release meant it was pinned rather than that it would let you update releases but not minor versions way back when we first added them).

Given that all our dependencies are < 1.0, I think it would make sense to specify releases for all of them. Want to add in the release numbers for the other three?